graph-oauth/src/accesstoken.rs

```diff
@@ -61,6 +62,17 @@ pub struct AccessToken {
     jwt: Option<JsonWebToken>,
 }
 
+use std::str::FromStr;
+use serde::de::Deserialize;
+fn from_str<'de, T, D>(deserializer: D) -> Result<T, D::Error>
```

Comment:
Still not sure this is needed. Can you give me an runnable example of this failing?